<launch>
  <env name="PYTHONPATH" value="$(env PYTHONPATH)" />

  <arg name="parent_ip" default="$(optenv PARENT_IP 127.0.0.1)" />
  <arg name="parent_port" default="$(optenv PARENT_PORT 9559)" />

  <!-- Publish camera and blob detection data. -->
  <include file="$(find nao_blob_detection)/launch/nao_blob_detection.launch" />
  
  <node pkg="nao_driver"
  		type="nao_controller.py"
  		name="nao_controller"
  		required="true"
  		args="--pip=$(arg parent_ip) --pport=$(arg parent_port)"
  		output="screen">
  		<!--<param name="init_stiffness" value="0.7" type="double" />-->
  </node>
  
  <node pkg="nao_experiment"
  		type="SetupGUI.py"
  		name="nao_experiment_setup_gui"
  		required="true"
  		args=""
  		output="screen" />
</launch>